pkgs/os-specific/linux/tuxedo-rs/default.nix

```diff
@@ -38,10 +52,11 @@ rustPlatform.buildRustPackage {
       - tailord: Daemon handling fan, keyboard and general HW support for Tuxedo laptops
       - tailor: CLI
     '';
-    homepage = "https://github.com/AaronErhardt/tuxedo-rs";
+    homepage = "https://github.com/${owner}/${name}";
```

Comment:
The way this was done previously is better, also because this doesn't typically change much.